packages/ember-views/lib/views/view.js

```diff
@@ -1022,6 +1022,8 @@ var View = CoreView.extend({
     @return Ember.View
   */
   nearestChildOf: function(klass) {
+    Ember.deprecate("nearestChildOf has been deprecated.\nFor existing behavior, use nearestOfType.");
```

Comment:
I'm open to an alternative. `For existing behavior` is pretty meaningless though. `For similar behavior` doesn't seem worth mentioning. Can you iterate a bit more?